### ISSUE TITLE
Some games duplicated

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/iyt#3]
I don't understand what's going on yet but some games in my history are being repeated.  I'll submit details later.

**Comment by pfuntner** [comment]
Some debugging focusing on a single game that is repeated:

```
$ ./history -vv 2>&1 | grep 15300001973767 
2019-04-07 08:28:12,131 DEBUG ./history:210 get_games_by_winloss Scraped from token 434/446, page 0/2: ('Regular', u'Backgammon', u'blondie_25419', u'15200000185645', u'15300001973767', 'Win', u'black', '00/07/25', u'14:19:00')
2019-04-07 08:28:12,160 DEBUG ./history:210 get_games_by_winloss Scraped from token 434/446, page 1/2: ('Regular', u'Backgammon', u'blondie_25419', u'15200000185645', u'15300001973767', 'Win', u'black', '00/07/25', u'14:19:00')
2019-04-07 08:28:12,186 DEBUG ./history:210 get_games_by_winloss Scraped from token 434/446, page 2/2: ('Regular', u'Backgammon', u'blondie_25419', u'15200000185645', u'15300001973767', 'Win', u'black', '00/07/25', u'14:19:00')
Regular,Backgammon,15300001973767,blondie_25419,15200000185645,Win,19,black,00/07/25,14:19:00
Regular,Backgammon,15300001973767,blondie_25419,15200000185645,Win,19,black,00/07/25,14:19:00
Regular,Backgammon,15300001973767,blondie_25419,15200000185645,Win,19,black,00/07/25,14:19:00
$ 
```

**Comment by pfuntner** [comment]
Another bug is that I was using `if not pages` to determine if I had to figure out whether I had to determine the number of pages or not.  I should have been using `if pages is None`.

**Comment by pfuntner** [comment]
I fixed this with 3f9891326465b7209fb343e7b6286c299336feaa

**Comment by pfuntner** [comment]
All of the reported games have a unique game id:

```
$ columns -c 3 < csv/20190407_102324_846368.csv  | uniqc | headtail
       1 39837 total items
       2     1   0.00% 15300001973767
       3     1   0.00% 15300001973776
       4     1   0.00% 15300001973786
       5     1   0.00% 15300001982935
         .
         .
         .
   39834     1   0.00% 15300066420983
   39835     1   0.00% 15300066420986
   39836     1   0.00% 15300066421061
   39837     1   0.00% 15300066437500
   39838     1   0.00% GameID
$
```

I have far less games now.  I thought I had around 55k but I have only around 39k.

My user profile says I've played:

- 19,660 Regular games
- 10,713 Tournament games
- 24,888 Ladder games

The total is 55,266 but I know that some of the games I've played do not appear on the summary pages.  I'm not sure that accounts for the difference or not.
